src/components/home/home.styles.js

```diff
@@ -107,28 +109,30 @@ const useHomeStyles = createUseStyles(theme => ({
       }
     },
     '& h3': {
-      color: '#000',
-      fontSize: '24px'
+      color: theme.palette.black,
+      fontSize: '23px',
```

Comment:
Why are some sizes specific numbers?

Reply:
Chami requested exactly those numbers, since she was not able to have multiples of 8 this time 😕

So you will see 23, 17, 29 etc. At one point I will redo them as we said, for example `theme.spacing(3) - 1`